packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -832,11 +839,19 @@ class RenderingEngine implements IRenderingEngine {
     // Deal with vtkjs driven viewports
     if (viewportInputEntries.length) {
       // 1. Getting all the canvases from viewports calculation of the new offScreen size
-
       const vtkDrivenCanvases = viewportInputEntries.map((vp) =>
         getOrCreateCanvas(vp.element)
       );
 
+      // Ensure the canvas size includes any scaling due to device pixel ratio
+      vtkDrivenCanvases.forEach((canvas) => {
```

Comment:
So it looks like you are doing the scaling here instead, can we nuke the scaling in my other comment above if its not needed?